gamla/functional_generic.py

```diff
@@ -428,6 +445,16 @@ def pipe(val, *funcs):
     lambda f: juxt(f, functional.second),
     before(functional.head),
 )
+
+#: Creates a function then maps the supplied mapper over the values of a dict.
+#:
+#:  >>> f = valmap(gamla.wrap_tuple)
```

Comment:
Consider using some arithmetical operator to make the example clearer (e.g. `gamla.add(1)`).
I read it twice to see the diff between `1` and `(1,)`